migrations/20180508154706_add_user_settings_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class AddUserSettingsTable extends AbstractMigration
+{
+    /**
+    * Migrate Up.
+    */
+    public function up()
+    {
+        $this->table('user_settings')
```

Comment:
This needs a unique key constraint ie. you can't have duplicate config_key's for the same user right?